README.md

```diff
@@ -83,6 +83,8 @@ Create a `profiles.clj` file in the project directory with the configuration set
 ```clojure
 {:profiles/dev
  {:env
+  ;; set migratus dir (and :migrations-dir) when developing with H2
```

Comment:
I think this needs to be expanded a bit. I'd not in the section above that you can either use H2 or Postgres, and give separate config examples for both cases.

Reply:
You are right, I guess I was a little bit too much brief about the usage. I was thinking about how it should be described and I see two options. 
1. For each DB type full individual sample differentiated only by the database setup. 
1. Individual smaller parts describing related part of the `profiles.clj` (currently three: PGSQL, H2, LDAP). 

Both have its ups and downs. 
1. Is little bit more chatty and worse for maintance (eg. LDAP gets something new), but copy paste friendly. 
1. Is easier for maitanance and easily extensible (eg. new supported DB), but requires some kind of ellipsis and forces developer to glue related parts together.

What would be your preference regarding this?

Reply:
From playing around with it locally, it looks like the only thing you have to change is the db URL with the new setup.  So, I'd opt for just changing the "Configuring PostgreSQL" database section into "Configuring the Database", and adding subsections for H2 and Postgres under it.